docs/modules/ROOT/pages/cops_rspec_factorybot.adoc

```diff
@@ -173,7 +175,7 @@ create_list :user, 3
 create_list :user, 3
 
 # good
-3.times { create :user }
+3.times.map { create :user }
```

Comment:
I'm not entirely convinced we should always recommend `.map`.
If the result is a standalone statement, `3.times { create(...) }` is fine.
But if it's inside a `let`:
```ruby
let(:three_models) { 3.times { create(...) }
```
or is assigned somewhere:
```ruby
three_models = 3.times { create(...) }
```
or used as an argument:
```ruby
create(:user, articles: 3.times { create(:book) })
```
`.map` is certainly essential.

In other cases - probably not.

Reply:
Yes, there will be cases where `.map` is not needed, but that seems something that must be given up for safer replacement.

I suppose I could add a conditional branch to reduce someunnecessary `.map` like `Lint/Void` Cop does:

1. `times` if it is on void context
2. `times.map` otherwise

However, it seems a bit too complicated, and it still does not work well for some common cases, such as the following:

```ruby
context 'when there are 3 users' do
  before do
    create_list(:user, 3)
  end

  # ...
end
```

```ruby
context 'when there are 3 users' do
  before do
    3.times.map { create(:user) }
  end

  # ...
end
```

Strictly speaking, the opposite pattern in create_list style can also cause problems:

```ruby
# before
count = 3.times { create(:user) }
puts "Created #{count} users"

# after (wrong correction)
count = create_list(:user, 3)
puts "Created #{count} users"
```